code/enumerable/inject-symbol-vs-block.rb

```diff
@@ -0,0 +1,24 @@
+require "rubygems"
+require "benchmark/ips"
+
+ARRAY = (1..1000).to_a
+
+def fastest
```

Comment:
I think naming the methods after their relative speed is a bit presumptuous. They're subject to change across runtimes and versions. I think it'd be better to name them after the type of operation being performed.